pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -57,3 +57,13 @@ type RemovePodsViolatingNodeAffinityArgs struct {
 	LabelSelector    *metav1.LabelSelector
 	NodeAffinityType []string
 }
+
```

Comment:
Missing `// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object` line

Reply:
Thanks a lot! `pull-descheduler-verify-master` can be passed now